src/SWP/Bundle/CoreBundle/Twig/ThemeExtension.php

```diff
@@ -89,8 +97,23 @@ public function getThemeLogoPath(string $fallBackPath): string
      * @param string $setting
      *
      * @return string
+     *
+     * @throws \Exception
      */
     public function getThemeSetting(string $setting): string
+    {
+        try {
+            return $this->getSetting($setting);
+        } catch (\Exception $e) {
+            if ('dev' !== $this->env) {
```

Comment:
maybe let's use here `'prod' ===`. As would be nice to catch those exceptions also in tests.